### ISSUE TITLE
[BUG] Site Improve Tool issue - Role with implied hidden content has keyboard focus

### ISSUE THREAD

**Issue by Satyam15302** [UKHO/admiralty-design-system#219]
## Describe the Bug

Certain elements have roles that imply their internal structure should be hidden from accessibility tools, meaning their child elements are treated as purely presentational. However, if a sub-element within such a role receives keyboard focus, it should be moved outside of the presentation role or the role should be adjusted.

## To Reproduce

1. Open the web application.

2. Navigate to the "Confirm exchange set content" Page.

3. Use a web accessibility testing tool ( siteimprove ) to identify the error.

## Expected Behavior

Focusable elements should not be placed inside parent elements that have roles suggesting their content should be hidden or presentational. Instead, such elements should be moved outside of these roles, or the parent role should be revised to allow interactive content.

## Screenshots

![Role with implied hidden content has keyboard focus](https://github.com/UKHO/admiralty-design-system/assets/108277912/0d2c74aa-e8d4-4e7d-a9bd-e30e0a79b9b8)

## Additional Context

In the application, there are instances where elements with implied presentational roles contain focusable child elements. This mismatch between the role and the focusable state can cause navigational and accessibility issues.

**Comment by KatiePUX** [comment]
duplicate of #213
